Zilon.Core/Zilon.Core.Specs/Steps/CommonSteps.cs

```diff
@@ -177,6 +170,7 @@ public CommonSteps(CommonGameActionsContext context) : base(context)
         {
             var playerState = Context.ServiceProvider.GetRequiredService<ISectorUiState>();
             var propTransferCommand = Context.ServiceProvider.GetRequiredService<PropTransferCommand>();
+            var gameLoop = Context.ServiceProvider.GetRequiredService<IGameLoop>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'gameLoop'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)